bundles/org.openhab.binding.hapero/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="Hapero" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>hapero Binding</name>
+	<description>Binding for data aquisition from hapero pellet stoves</description>
+
+</binding:binding>
```

Comment:
Please convert to the new `addon.xml`.

Reply:
Related issue is here: https://github.com/openhab/openhab-core/issues/2058
But you can also update your branch and look at another binding's `addon.xml` and adapt it to this initial contribution.

Reply:
OK, done. 
Since i accidentally merged the upstream at one point instead of rebasing (i fixed that with a force push) i now have 100 or so review requests and dont know how to remove them. But otherwise it seems OK

Reply:
Removed the excess

Reply:
Thanks @lsiepel !